sklearn/metrics/scorer.py

```diff
@@ -58,13 +106,43 @@ def __repr__(self):
                    "" if self._sign > 0 else ", greater_is_better=False",
                    self._factory_args(), kwargs_string))
 
+    def __call__(self, estimator, X, y_true, sample_weight=None):
+        """Evaluate predicted target values for X relative to y_true.
+
+        Parameters
+        ----------
+        estimator : object
+            Trained estimator to use for scoring. Must have a predict_proba
+            method; the output of that is used to compute the score.
+
+        X : array-like or sparse matrix
+            Test data that will be fed to estimator.predict.
+
+        y_true : array-like
+            Gold standard target values for X.
+
+        sample_weight : array-like, optional (default=None)
+            Sample weights.
+
+        Returns
+        -------
+        score : float
+            Score function applied to prediction of estimator on X.
+        """
+        return self._score(estimator, X, y_true, sample_weight=sample_weight)
+
+    def _method_cacher(self, estimator, method, *args, **kwargs):
```

Comment:
I'm confused that this is called `_method_cacher`. Makes me things that it overrides the `MultimetricScorer`'s `_method_cacher`, but the logic is slightly different.

Call this `_passthrough_method_cacher`?